packages/dice-webapp/components/admin/organisation-req-card.tsx

```diff
@@ -23,48 +42,106 @@ export const OrganisationReqCard = ({
   onAccept,
   onReject,
   className,
+  institutionDetails,
 }: OrganisationReqCardProps) => {
   const { contract } = useWeb3();
+  const [formData] = useState({
```

Comment:
why use a useState for constant data? remove it and directly use the parameters in the component